Editor/Window/Components/ProfileSelector.cs

```diff
@@ -26,19 +26,19 @@ public ProfileSelector()
 
             LocalizeText();
 
-            _stateManager = EditorWindow.GetWindow<GameLiftPlugin>().StateManager;
+            _stateManager = EditorWindow.GetWindow<GameLiftPlugin>()._stateManager;
             _stateManager.OnProfileSelected += UpdateGUI;
             _textProvider = TextProviderFactory.Create();
 
-            _dropdown.RegisterValueChangedCallback(value => { _stateManager.SelectedProfile = value.newValue; });
+            _dropdown.RegisterValueChangedCallback(value => { _stateManager.SelectedProfileName = value.newValue; });
```

Comment:
We have a list of `UserProfiles` in `GameLiftSettings.yaml`, and this is selected profile name and we use it to index the list of profiles for the correct one. 
We already have a `SetProfile(string profileName)` method but it is private and used as a setter method of `SelectedProfileName`. Will look at moving this to public and using that instead of a public property `SelectedProfileName`